docs/operations/overlapping-network-ranges.md

```diff
@@ -0,0 +1,45 @@
+# Overlapping Network Ranges
+
+## Motivation
+Shoot clusters, and their control planes are deployed in a network managed by the Gardener. The control plane can
+access shoot pods, services and nodes via a VPN connection. This makes it necessary that both the shoot and the 
+control plane network ranges do not overlap. This limitation can be problematic in some cases where the network 
+requirements of the shoot make a placement on existing seed clusters impossible. In the past, this has led to the costly
+mitigation of deploying additional seed clusters with non-overlapping network ranges.
+This document describes how Gardener handles overlapping network ranges in shoot clusters, 
+which allows for more flexibility in network design and shoot placement.
+
+## How it works
+Inside the VPN tunnel, seed pod IPs and shoot IPs are disentangled by using a double-NAT mechanism, 
+i.e., both the source and destination IPs are translated as shown in the following diagram:
+
+![double-nat](./images/double-nat.png)
+
+The NAT ranges are reserved from the 240/4 range as defined in [RFC 6890](https://datatracker.ietf.org/doc/html/rfc6890#section-2.7.4)
+and thus cannot be used as shoot ranges. You can find the specific ranges in the [user networking guide](https://github.com/gardener/gardener/blob/master/docs/usage/networking/shoot_networking.md#reserved-networks)
+
+> [!IMPORTANT]
+> The double-NAT mechanism only applies to IPv4 networks. For dual-stack or IPv6-only shoots, the shoot CIDRs must not overlap with the seed CIDRs. However, since the IPv6 range is usually assigned by the cloud provider, this is not a problem in practice.
```

Comment:
This sentence sounds confusing to me. In case of dual-stack IPv4 addresses can't overlap?